packages/devtools_app/lib/src/framework/app_error_handling.dart

```diff
@@ -86,3 +107,55 @@ void reportError(
     );
   }
 }
+
+SingleMapping? _cachedJsSourceMapping;
+SingleMapping? _cachedWasmSourceMapping;
+
+Future<SingleMapping?> _fetchSourceMapping() async {
+  final cachedSourceMapping = preferences.wasmEnabled.value
```

Comment:
instead of using the wasm preference, use `kIsWasm` since this holds the true value of whether wasm is actually being used or not

Reply:
Done!